### ISSUE TITLE
Build for Debian 12 bookworm 

### ISSUE THREAD

**Issue by Ninofj** [kasmtech/KasmVNC#163]
A couple of days ago Debian 12 "bookworm" was released. Is there a build planned for the new Debian version or is it too soon? 

Best regards

**Comment by tony1016** [comment]
SUPPORT!!

**Comment by 0x09AF** [comment]
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm

$ sudo apt install ./kasmvncserver_bullseye_1.1.0_arm64.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'kasmvncserver' instead of './kasmvncserver_bullseye_1.1.0_arm64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 kasmvncserver : Depends: libssl1.1 (>= 1.1.0) but it is not installable
E: Unable to correct problems, you have held broken packages.

$ apt list --installed | grep libssl
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
libssl3/stable,now 3.0.9-1 arm64 [installed]
```

**Comment by mmcclaskey** [comment]
Here is a pre-release build for Bookworm. We should be making an official release this month.

[KasmVNC Bookworm amd64](https://kasmweb-build-artifacts.s3.amazonaws.com/kasmvnc/fa884809145e0ba6e5e1da49887a82b5cab0a1d1/kasmvncserver_bookworm_1.1.1_master_fa8848_amd64.deb)
[KasmVNC Bookworm arm64](https://kasmweb-build-artifacts.s3.amazonaws.com/kasmvnc/fa884809145e0ba6e5e1da49887a82b5cab0a1d1/kasmvncserver_bookworm_1.1.1_master_fa8848_arm64.deb)

**Comment by ich777** [comment]
I think this issue can be closed @Ninofj because the version 1.2.0 includes a build for `bookworm`.
